tests/Coverage/index.test.ts

```diff
@@ -1,9 +1,10 @@
 jest.disableAutomock()
+import * as Coverage from '../../src/Coverage'
 
 describe('Coverage', () => {
   describe('module.exports', () => {
     it('should match the snapshot', () => {
-      expect(require('../../src/Coverage')).toMatchSnapshot()
+      expect(Coverage).toMatchSnapshot()
```

Comment:
Feel free to remove this test and the snapshot if you want. Why did I add a snapshot test of the API?! Ick.
(Also, we can also defer that.)

Reply:
cool, removed.